### ISSUE TITLE
Warning: Illegal offset type in isset or empty

### ISSUE THREAD

**Issue by zonky2** [heimrichhannot/contao-multi-column-editor-bundle#7]
Warning: Illegal offset type in isset or empty

```
ErrorException:
Warning: Illegal offset type in isset or empty

  at /home/xxx/contao4_2020/vendor/heimrichhannot/contao-multi-column-editor-bundle/src/EventListener/HookListener.php:94
  at HeimrichHannot\MultiColumnEditorBundle\EventListener\HookListener->loadDataContainerHook('tl_files')
     (/home/xxx/contao4_2020/vendor/contao/core-bundle/src/Resources/contao/library/Contao/DcaLoader.php:119)
  at Contao\DcaLoader->loadDcaFiles(false)
     (/home/xxx/contao4_2020/vendor/contao/core-bundle/src/Resources/contao/library/Contao/DcaLoader.php:70)
  at Contao\DcaLoader->load(false)
     (/home/xxx/contao4_2020/vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:1331)
  at Contao\Controller::loadDataContainer('tl_files')
     (/home/xxx/contao4_2020/vendor/contao/core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php:372)
  at Contao\DcaExtractor->createExtract()
  ...
```

**Comment by zonky2** [comment]
fix

```diff
        if (!($name = System::getContainer()->get('huh.request')->getPost('name'))) {
            return;
        }

+        if (is_array($name)) {
+            return;
+        }

        if (isset($dca['fields'][$name])) {
            return;
        }
```

**Comment by zonky2** [comment]
pls update or fix the repo

**Comment by Defcon0** [comment]
Thanks, @zonky2 !

**Comment by Defcon0** [comment]
Merged in v2.5.3
